SPECS/libhif/libhif.spec

```diff
@@ -36,16 +39,20 @@ used by PackageKit and rpm-ostree.
 %package devel
 Summary: GLib Libraries and headers for libhif
 Requires: libhif
+Provides: pkgconfig(libhif)
+Provides: pkgconfig(hawkey)
+Provides: pkgconfig(rpm)
```

Comment:
This packages should not Provide pkgconfig(rpm) and pkgconfig(hawkey) etc. They should be provided by respective packages SPEC files.